.github/workflows/test.yml

```diff
@@ -1,5 +1,6 @@
 name: Test
 on:
+  workflow_dispatch:
```

Comment:
Checks were failing on the main branch for me locally (??). I've added the ability to manually trigger tests to be able to sense check things like this.